agent/connect/ca/provider_aws.go

```diff
@@ -33,9 +33,6 @@ const (
 	// leaf cert.
 	LeafTemplateARN = "arn:aws:acm-pca:::template/EndEntityCertificate/V1"
 
-	// RootTTL is the validity duration for root certs we create.
-	AWSRootTTL = 5 * 365 * 24 * time.Hour
-
 	// IntermediateTTL is the validity duration for the intermediate certs we
 	// create.
 	AWSIntermediateTTL = 1 * 365 * 24 * time.Hour
```

Comment:
probably in a subsequent PR, but I think we can also make this configurable.

Reply:
the AWSRootTTL is actually replaced by the RootCertTTL option :)